redisinsight/api/src/modules/redis/client/node-redis/cluster.node-redis.client.ts

```diff
@@ -49,10 +49,13 @@ export class ClusterNodeRedisClient extends NodeRedisClient {
   /**
    * @inheritDoc
    */
-  async sendPipeline(commands: RedisClientCommand[]): Promise<Array<[Error | null, RedisClientCommandReply]>> {
+  async sendPipeline(
+    commands: RedisClientCommand[],
+    options?: IRedisClientCommandOptions,
+  ): Promise<Array<[Error | null, RedisClientCommandReply]>> {
     return Promise.all(
       commands.map(
-        (cmd) => this.sendCommand(cmd)
+        (cmd) => this.sendCommand(cmd, options)
```

Comment:
As I remember we must have some kind of `prepareCommandOptions` method to make our custom options match with native lib options. please use it

Reply:
Pipeline calls a sendCommand that has preparation of options 
`return this.client.sendCommand(
      NodeRedisClient.prepareCommandArgs(command),
      NodeRedisClient.prepareCommandOptions(options),
    );`

Reply:
agree. forgot about it